execution/block-partitioner/src/pre_partition/connected_component/config.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright © Aptos Foundation
+
+use crate::pre_partition::{
+    connected_component::ConnectedComponentPartitioner, PrePartitioner, PrePartitionerConfig,
+};
+
+#[derive(Clone, Debug)]
+pub struct ConnectedComponentPartitionerConfig {
+    pub load_imbalance_tolerance: f32,
+}
+
+impl Default for ConnectedComponentPartitionerConfig {
+    fn default() -> Self {
+        ConnectedComponentPartitionerConfig {
+            load_imbalance_tolerance: 4.0,
+        }
+    }
+}
+
+impl PrePartitionerConfig for ConnectedComponentPartitionerConfig {
+    fn build(&self) -> Box<dyn PrePartitioner> {
```

Comment:
Do we need Box for PrePartitioner ? It has only one owner and has no dynamically variable size. We could just create the PrePartitioner obj and move it to the caller ?

Reply:
Do you mean `fn build(&self) -> PrePartitioner`?
But `PrePartitioner` is a trait, and I'm not aware of a way to return a trait...

Or maybe I misunderstood. Can you create a sub-PR?

Reply:
Oh ok, I missed the trait part. We need Box here